templates/fine-tune-llm_v2/cookbooks/continue_from_checkpoint/README.md

```diff
@@ -0,0 +1,74 @@
+# Continue fine-tuning from a previous checkpointing
+
+**⏱️ Time to complete**: 40 minutes
+
+This guide assumes that you have familiarized yourself with the main fine-tuning guide of this template.
+In this cookbook tutorial, we showcase how a checkpoint that was created earlier can be used as initialization for another round of fine-tuning.
+This allows us to sequentially combine fine-tuning on multiple datasets in order to get performance boost on the final task that we care about. 
+
+We support both Full-parameter checkpoints, and LoRA-adapter checkpoints. However, we recommend not combining the two by training a full-parameter model followed by a LoRA adaptation, because serving the resulting LoRA adapter will require the full-parameter checkpoint. Unless you are fine-tuning many such LoRA adaptors for different tasks this serving architecture does not have the neither the economical benefits of LoRA nor the quality benefits of full-parameter.
+
+## How to fine-tune from a previous checkpointing
+
+
+```python
+# [Optional] You can set the WandB API key to track model performance
+# !export WANDB_API_KEY={YOUR_WANDB_API_KEY}
+
+# Continue LoRA fine-tuning on the GSM8k dataset with Llama 3 8B
+!llmforge dev finetune llama-3-8b.yaml
+```
+
+## What we are fine-tuning on
```

Comment:
Let's remove this title and continue explaining what happens in the code-block in the next paragraph.

Reply:
You mean remove the entire block of "What are we fine-tuning on"?

Reply:
No just the title.